Project.toml

```diff
@@ -45,9 +45,10 @@ JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
 MultivariatePolynomials = "102ac46a-7ee4-5c85-9060-abc95bfdeaa3"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Polyhedra = "67491407-f73d-577b-9b50-8179a7c68029"
```

Comment:
Should this be in the docs project only?

Reply:
This may be a mistake on my part, I'm not sure

Reply:
it was originally only in the docs project

Reply:
```suggestion
```
Then i guess we remove it!